avaframe/com1DFA/com1DFA.py

```diff
@@ -2093,15 +2097,21 @@ def DFAIterate(cfg, particles, fields, dem, inputSimLines, outDir, cuSimName, si
     t = particles["t"]
     log.debug("Saving results for time step t = %f s", t)
 
-    # export initial time step
-    if cfg["EXPORTS"].getboolean("exportData"):
+    # Initialize particles output directory if needed
+    if "particles" in resTypes:
+        outDirData = outDir / "particles"
+        fU.makeADir(outDirData)
```

Comment:
why outside of exportData if?

Reply:
Because the if below only covers the initial timestep export. If that one is not hit, the next (intermediate and final) would need the  same check. To keep it simple, I added it here.